File: src/entity/enemy/SimpleEnemy.java
package entity.enemy;

import javafx.scene.image.Image;

public class SimpleEnemy extends Enemy {
	
	public static final String IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile245.png";
	
	public SimpleEnemy(int x, int y) {
		super(x, y, 5, 1);
		img = new Image(IMAGE_URL);
		setDir('E');
	}
	
}

File: src/entity/enemy/SimpleEnemy.kt
package entity.enemy

import javafx.scene.image.Image

class SimpleEnemy(x: Int, y: Int) : Enemy(x.toDouble(), y.toDouble(), 5, 1.0) {
    override var img: Image = Image(IMAGE_URL)

    init {
        setDir('E')
    }

    companion object {

        val IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile245.png"
    }

}
